### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Eternalgratis**
Pair to pair programming on mobile menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -42,8 +42,86 @@ ul {
...
+  height: 100vh;
+  gap: 0;
+  width: 100%;
+  text-align: start;
+  flex-direction: column;
+  transition: 0.3s;
+  background-color: rgb(248, 250, 249);
+  padding-left: 40px;
+  padding-top: -10%;
+}
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, kindly add the missing cancel button to the mobile menu to match the Figma design.
- You could add a `z-index: 1` to make the `.nav-menu` covers other element and so the height takes up the whole device height.

x

File: main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+// hamburger.addEventListener("click", () => {
+//   hamburger.classList.toggle("active");
+//   navMenu.classList.toggle("active");
+// })
+
+//  document.querySelectorAll(".nav-link").forEach((n) => n.addEventListener("click", () => {
+//    hamburger.classList.remove("active");
+//    navMenu.classList.remove("active");
+//  }))
```
Comment (zieeco) [inline main.js]: **[REQUIRED CHANGES]**

- Please, kindly remove this commented code block, they make your code base not clean and professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @Eternalgratis, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :zap: 

- Nice Design :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Eternalgratis        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**